### DISCUSSION TITLE
bb not fetching all my postgresql tables

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6414] [Help] [answered]
I have 5 tables in my schema but bb is only fetching 3 of these tables.. Is there a known issue ?
Nothing weird in the 2 remaining tables...

**Comment by melohagan** [comment, marked as answer]
Hey @NicolasFabre81 

Would you be able to provide the schemas for the remaining tables? 

Are they definitely in the same DB and schema?

**Reply by NicolasFabre81** [reply]
Found the solution : it's because these tables do not have a primary key...
